docs/references/protocol/transactions/types/mptokenissuancecreate.md

```diff
@@ -54,4 +55,13 @@ Transactions of the MPTokenIssuanceCreate type support additional values in the
 | `tfMPTCanTransfer` | `0x00000020` | `32`          | If set, indicates that tokens can be transferred to other accounts that are not the issuer. |
 | `tfMPTCanClawback` | `0x00000040` | `64`          | If set, indicates that the issuer can use the `Clawback` transaction to claw back value from individual holders. |
 
+## Error Cases
+
+Besides errors that can occur for all transactions, {% $frontmatter.seo.title %} transactions can result in the following [transaction result codes](../transaction-results/index.md):
+
+| Error Code                | Description |
+|:--------------------------|:------------|
+| `temDISABLED`             | Either the `PermissionedDomains` or `SingleAssetVault` amendment is disabled. |
+| `temMALFORMED`            | The `DomainID` is `0` or empty; you can also receive this error if you include a `DomainID` without setting the `tfMPTRequireAuth` flag. |
```

Comment:
I see some other error codes that can occur [in the source](https://github.com/XRPLF/rippled/blob/develop/src/xrpld/app/tx/detail/MPTokenIssuanceCreate.cpp#L29-L111), such as `tecINSUFFICIENT_RESERVE` and `temBAD_TRANSFER_FEE`.

Reply:
I am not sure how the reader will interpret `is empty`. The `DomainID` is an optional field, many users might not even want to use it. **IF** they do use it, it must point to a valid permissioned domain.

Reply:
Also, there are other possible reasons to get `temMALFORMED` error